src/eventHandlers/messageBroker.ts

```diff
@@ -114,6 +114,7 @@ export function createPostToRabbitMQHandler({
           proposalBookingId: scheduledevent.proposalBookingId,
           status: scheduledevent.status,
           proposalPk: proposalBooking?.proposal.primaryKey,
+          localContactId: scheduledevent.localContact?.id || null,
```

Comment:
Just a small note ?? is more appropriate here than ||, because there could be a local contact with ID 0. Then you want the value to be 0, not null :)